docs/sql-reference/sql-statements/data-definition/ALTER MATERIALIZED VIEW.md

```diff
@@ -2,12 +2,16 @@
 
 ## Description
 
-Alters the name or refresh strategy of an asynchronous materialized view.
+This SQL statement can：
```

Comment:
1. use complete sentences for the three items, eg. "Alter the name of a materialized view." 2. spelling, "materialized" in the first item